README.md

```diff
@@ -563,14 +563,7 @@ end
 Then you can get this error message in exception handler:
 ```ruby
 rescue_from Pundit::NotAuthorizedError do |e|
-  message = 
-    if e.reason 
-      I18n.t("pundit.errors.#{e.reason}", scope: "pundit", default: :default)
-    else 
-      e.message
-    end
-
-  flash[:error] = message
+  flash[:error] = I18n.t("errors.#{e.reason}", scope: "pundit", default: e.message)
```

Comment:
Is there a special reason to use `scope` here? If not then can we keep this example as simple and conventional as possible by using the basic syntax: `pundit.errors...`

Reply:
No there is no special reason for it. Here I just tried to follow current repo style, and that I found in above readme section https://github.com/varvet/pundit/blob/master/README.md#creating-custom-error-messages

Reply:
Ah I see: you're absolutely right. I don't like it, but it's consistent. Let's leave the `scope`.